src/components/search/MovieCard.js

```diff
@@ -1,43 +1,62 @@
 import React from 'react'
 import styled from 'styled-components'
+import { Link } from 'react-router-dom'
 
-const MovieCard = ({ imgSrc, year, title, category, id, className }) => {
+const MovieCard = ({ className, release_year, show_title, category }) => {
+  const href = `/film/${show_title}`
```

Comment:
Don't see a reason to use a constant for that path here, but let's keep it here for usability